modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllCheck.java

```diff
@@ -407,12 +442,38 @@ private String _formatParameters(
 				getVariableName(methodCall), CharPool.PERIOD, newMethodCall);
 		}
 
-		newMethodCall = _addOrReplaceParameters(
-			newMethodCall, parameterNames, JavaSourceUtil.getParameterList(to));
+		newMethodCall = _addOrReplaceMethodParameters(
+			parameterNames, newMethodCall, JavaSourceUtil.getParameterList(to));
 
 		return StringUtil.replaceFirst(newContent, methodCall, newMethodCall);
 	}
 
+	private String _formatTypeParameters(
+		String methodCall, String newContent, String to) {
+
+		String newMethodCall = methodCall.substring(
+			0, methodCall.indexOf(CharPool.LESS_THAN) + 1);
+
+		String typeParameterName = methodCall.substring(
+			methodCall.indexOf(CharPool.LESS_THAN) + 1,
+			methodCall.lastIndexOf(CharPool.GREATER_THAN));
+
+		List<String> typeParameterNames = Arrays.asList(
+			typeParameterName.split(StringPool.COMMA_AND_SPACE));
+
+		String newTypeParameterName = to.substring(
+			to.indexOf(CharPool.LESS_THAN) + 1,
+			to.lastIndexOf(CharPool.GREATER_THAN));
+
+		List<String> newTypeParameterNames = Arrays.asList(
+			newTypeParameterName.split(StringPool.COMMA_AND_SPACE));
```

Comment:
@TamyrisTorres, please declare the variables according to their usage, e.g. the `newTypeParameterNames` is used first than `typeParameterNames`.